From 71115bc6e70254cc857fb75739b97112b9015eff Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Wed, 23 Mar 2016 16:01:32 +0100
Subject: [PATCH 3/3] build with libtirpc and gcc5

This is a workaround to build library with libtirpc and gcc5
---
 bindings/java/hyperic_jni/jni-build.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/bindings/java/hyperic_jni/jni-build.xml b/bindings/java/hyperic_jni/jni-build.xml
index d4556c3..eeb95ff 100644
--- a/bindings/java/hyperic_jni/jni-build.xml
+++ b/bindings/java/hyperic_jni/jni-build.xml
@@ -324,6 +324,8 @@
         <compilerarg value="-O2" if="jni.optim"/>
         <compilerarg value="-g" if="jni.debug"/>
         <compilerarg value="-Wall"/>
+        <compilerarg value="-std=gnu89"/>
+        <compilerarg value="-I/usr/include/tirpc"/>
         <compilerarg value="-Werror" if="jni.werror"/>
         <compilerarg value="${jni.gccm}" if="jni.gccm"/>
         <defineset>
@@ -341,6 +343,7 @@
                 dir="${jni.dmalloc}/lib"
                 libs="dmallocth"/>
         <linkerarg value="${jni.gccm}" if="jni.gccm"/>
+	<libset libs="tirpc "/>
       </linker>
 
       <!-- Darwin -->
-- 
2.8.1

